plugins/coverimage.koplugin/main.lua

```diff
@@ -193,6 +228,58 @@ function CoverImage:addToMainMenu(menu_items)
                     end
                 end,
             },
+            -- menu entry: scale book cover
+            {
+                text = _("Scale book cover"),
+                help_text = _("If disabled, the cover will have to be scaled by the FW.\nDisabling is the fastest method, but doesn't work on all devices."),
```

Comment:
Haven't seen how this is presented, but it seems a bit strange:
`text = _("Scale book cover"),` hints it might be a toggle, or some scaling options
`sub_item_table ` presents "black|white|grey background" options, which seem totally unrelated to scaling :) 
`help_text` is then not helpfull, mentionning "disabling" which one can't find anywhere :/

Reply:
Should I rename `Scale book cover` to `Scale book cover and add background` and
the help text to:   _("If a background color is selected, the image will be scaled with its original aspect ratio and a solid backround to fit the device screen.\nIf no background color is selected, the cover is saved in the original aspect ratio and it has to be scaled by the FW.\nThe latter is the fastest method, but doesn't work on all devices.")?

Reply:
We use the word "Stretch" in the regular screensaver - it sounds clearer than "Scale" in expressing that the image will be distorted :) 
May be you should have 4 items instead of 3, mutually exclusive: `[ ] Strech to fit screen`, with always one checked.
Your main menu title could then be `Stretching and background`.